### PR TITLE
EPMRPP-38514, EPMRPP-38515: Part of EPMRPP-33317 add integration with BE for add/edit defect type actions (without validation)

### PR DESCRIPTION
**Pull request by margalencia**
Add and Edit action for Defect Types. Validation will be in the next PR.

### CODE REVIEW

File: app/src/controllers/project/sagas.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import { showNotification, NOTIFICATION_TYPES } from 'controllers/notification';
+import { activeProjectSelector } from 'controllers/user';
+import { fetch } from 'common/utils';
+
+import { UPDATE_DEFECT_SUBTYPE, ADD_DEFECT_SUBTYPE } from './constants';
+import { updateDefectSubTypeSuccessAction, addDefectSubTypeSuccessAction } from './actionCreators';
+
+function* updateDefectSubType({ payload: subType }) {
+  try {
+    const activeProject = yield select(activeProjectSelector);
```
Comment (AmsterGet) [inline app/src/controllers/project/sagas.js]: Please use `projectIdSelector` from `controllers/pages` to successfully send requests from project settings page on Admin part.

File: app/src/controllers/project/sagas.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  }
+}
+
+function* watchUpdateDefectSubType() {
+  yield takeEvery(UPDATE_DEFECT_SUBTYPE, updateDefectSubType);
+}
+
+function* addDefectSubType({ payload: subType }) {
+  try {
+    const activeProject = yield select(activeProjectSelector);
```
Comment (AmsterGet) [inline app/src/controllers/project/sagas.js]: The same as in comment above.

File: app/src/controllers/project/sagas.js (hunk 1 above)
Comment (margalencia) [reply app/src/controllers/project/sagas.js]: done

File: app/src/controllers/project/sagas.js (hunk 2 above)
Comment (margalencia) [reply app/src/controllers/project/sagas.js]: done

**Pull request merged**